.github/workflows/build_test.yaml

```diff
@@ -64,6 +64,25 @@ jobs:
       run: cmake --build build -j4
     - name: run tests
       run: cd build/ && ctest -j4 -C Debug --output-on-failure
+  windows-shared-libs-build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: vcpkg build
+      uses: johnwason/vcpkg-action@v6
+      id: vcpkg
+      with:
+        pkgs: jsoncpp eigen3 curl libtins glfw3 glew spdlog libpng flatbuffers gtest
+        triplet: x64-windows
+        revision: 2024.04.26
+        token: ${{ github.token }}
+        github-binarycache: true
+    - name: cmake configure
+      run: cmake . -B build -DBUILD_TESTING=ON -DBUILD_EXAMPLES=ON -DBUILD_SHARED_LIBS=ON -DCMAKE_TOOLCHAIN_FILE=D:/a/ouster-sdk/ouster-sdk/vcpkg/scripts/buildsystems/vcpkg.cmake
```

Comment:
What is `D:/a`?

Reply:
Just an absolute path which I had to use to get this to work, see the step above.

Reply:
We could probably parameterize the `BUILD_SHARED_LIBS` rather than duplicating the build step.

Reply:
Agreed. I'm not too familiar with this CI, sorry for my shabby solution.